### PR TITLE
Translation of Cloud Native Security White Paper(Chinese) #470

### PR DESCRIPTION
**Pull request by babysor**
According to proposal #470 ,  we conducted a Chinese translation and review of Cloud Native Security White Paper. Thanks to the amazing teamwork and Jimmy's initiating.

### CODE REVIEW

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 1]
```diff
@@ -0,0 +1,825 @@
+# 云原生安全白皮书中文版
+
+[TOC]
+
+
+
+# **执行摘要**
```
Comment (rootsongjc) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:7]: 标题里的加粗格式可以去掉。

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+
+开发人员，操作员和安全人员应针对关键业务，具有高威胁特征，易受频繁更改或具有错误历史记录的代码和基础结构构建测试。威胁建模可以识别高风险和高影响力的代码热点，针对这些热点开发测试可提供高投入回报（ROI）。测试可能包括部署，操作系统，基础架构和数据库强化，应用程序测试（静态和动态源代码测试，容器配置），集成或系统测试（接受应用程序和基础架构组件及其交互）以及冒烟测试（部署后）检查实时系统）。测试作者应该可以访问全面的开发和测试环境，从而使他们能够执行快速的测试开发，同时减少持续集成（CI）反馈循环。系统测试套件应可供作者在本地以及内部运行。
+
+### **代码审查**
+
+对工作负载或已部署工作负载的基础结构进行的细微更改可能会产生深远的安全后果。 为了减轻意外后果的风险，鼓励团队在将先前的更改合并到代码库中之前（例如，在 git 工作流中实现拉取请求）在进行代码审查时使用“四眼”原则。
+
+## **分发**
+
+![img](https://docimg3.docs.qq.com/image/HrNJkDYdc9qke6Cgq6wEGg?w=1223&h=591)
```
Comment (rootsongjc) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:145]: 图片应该是使用 GitHub 里的相对链接。

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 3]
```diff
@@ -0,0 +1,825 @@
...
+
+IaC 越来越受欢迎，在企业中的部署迅速增加，以部署云和容器基础设施。因此，这些模板中的不安全配置可能导致其成为被攻击目标。
+
+在部署应用程序和基础设施组件之前，应使用自动化工具扫描这些模板，以发现不安全的配置和其他安全控制漏洞。需要注意的关键错误配置包括：
+
+●   应用程序清单中指定的镜像中包含的漏洞。
+
+●   配置设置，比如可以升级权限的容器。
+
+●   识别可能危及系统的安全上下文和系统调用；
```
Comment (rootsongjc) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:211]: 列表格式需要调整。

**Comment by Gsealy** [comment]
@rootsongjc @babysor 
about ***Encryption*** part, I have retranslate it try to read more fluently.
PTAL

```markdown
#### 加密

对容器镜像进行加密，从而保证镜像内容的机密性。加密后，以确保它们从构建到运行前都是密文态。当加密镜像分发后受到破解，制品库中存储的镜像仍然是加密的，这有助于保护商业机密或其他保密材料等。

容器镜像加密的另一个常见用途是容器镜像授权的增强。当镜像加密与密钥证明管理和/或授权、认证发布等相结合时，可以要求容器镜像只能在特定平台上运行。容器镜像授权也适合保证合规性，例如地理限制、出口管控和数字版权媒体管理。
```

**Comment by rootsongjc** [comment]
> @rootsongjc @babysor
> about _**Encryption**_ part, I have retranslate it try to read more fluently.
> PTAL
> 
> ```gfm
> #### 加密
> 
> 对容器镜像进行加密，从而保证镜像内容的机密性。加密后，以确保它们从构建到运行前都是密文态。当加密镜像分发后受到破解，制品库中存储的镜像仍然是加密的，这有助于保护商业机密或其他保密材料等。
> 
> 容器镜像加密的另一个常见用途是容器镜像授权的增强。当镜像加密与密钥证明管理和/或授权、认证发布等相结合时，可以要求容器镜像只能在特定平台上运行。容器镜像授权也适合保证合规性，例如地理限制、出口管控和数字版权媒体管理。
> ```

可以使用这个翻译替换下。

**Comment by lumjjb** [comment]
Thanks @babysor , @rootsongjc, @Gsealy! This is really great! I think that if there is a consensus among the 3 of you that it is good, then we can go ahead and approve this.

I would add a couple asks:
- Can we add a link to the version that is being translated. Want to make sure that we keep that information in case we need to make edits to the whitepaper that need to be propagated. We can link to a commit ID of the github repo.
- For the authors, seems like there's a mix, I think let's keep it all the same or add brackets for translation of names.

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 4]
```diff
@@ -0,0 +1,828 @@
...
+
+# 执行摘要
+
+## 宗旨
+
+整个技术行业正在向"云原生"的开发和部署模式转变，相关的技术、产品、标准和解决方案等生态系统也在不断扩大，这对于决策者来说，面对新的复杂设计，如何跟上这个步伐将是个巨大的挑战。尤其对于 CISO 的角色，在这个高速变化的环境中，同时肩负着阐明业务价值主张的责任。云原生安全也非常鼓励与现行的其他实践结合，如 scrum 敏捷、DevOps 等工作流程，以便于更好的改变和加速原有的处理模式。
+
+### 问题分析
+
+随着技术的进步，我们所处的网络环境正日益复杂，安全风险也不断升级。传统基于边界的安全防护机制，如防火墙等，在云大物移的场景下很难面面俱到。因此，如果只关注云原生带来的开发和部署的敏捷性，我们或将直面安全风险。为更好保护我们的应用，我们需要一些新的安全方法，更接近于基于元数据或属性来识别动态工作负载（如 SDP 软件定义边界提供了一种实现思路）。这类方法可以更细粒度的识别和保护工作负载，以满足云原生应应用快速扩展、弹性伸缩的特性以及不断变化的外界环境威胁。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:15]: 云大物移 -> 云原生

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 5]
```diff
@@ -0,0 +1,828 @@
...
+
+这些模式的转变要求在应用安全生命周期中采用更自动化和安全的设计架构（如零信任）。容器化作为云原生环境的核心转变之一，也需要新的最佳实践。实施安全策略将影响到组织内的多个利益相关方，并极大地影响开发和运维人员的生产力。云原生应用仍然需要开发、分发、部署和运维，但其模式决定了新的安全机制，通过这些机制可以有效地实现这些目标。
+
+这些模式的转变，需要在应用安全生命周期中采用更自动化和安全的架构设计（如零信任）。实施安全策略的过程中，将受到到组织内的多个利益相关方的影响，比如可能会影响开发和运维人员的生产力。成功的安全实施需要考虑在安全与效率间取得平衡。因此，云原生应用仍会经历研发、分发、部署和运维等环节，研发模式决定了新的安全机制的实施空间，而安全机制可以有效地实现应用的安全目标。
+
+云原生开发可以采用不同的方式来建模，来构成应用开发的生命周期，研发、分发、部署、运行态是其中比不可少的要素之一。云原生安全与传统安全方法的最大不同在于，云原生安全需要有一种强有力的机制，更柔和的嵌入方式，来确保安全措施在不同的环节中有效实施。而不能像传统安全方法一样在应用开发生命周期的末尾，通过分散且粗暴的方式进行干预管理。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:21]: 比不可少 -> 必不可少

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 6]
```diff
@@ -0,0 +1,828 @@
...
+
+本文档旨在为组织以及技术领导者提供对云原生安全的清晰理解，及其如何在参与整个生命周期流程中使用和评估安全相关的最佳实践。云原生安全是一个多目标和多限制的复杂问题范畴，会跨越许多专业技术和实践领域。软件生命周期中 Day 1、Day 2 的绝大多数操作都会涉及到从身份管理到存储解决方案的安全技术或领域。然而，云原生安全所涵盖的内容远不止这些领域；它是个关于人的广义问题范畴，包含个体、团队和组织。它应该成为人和系统在深度使用甚至改造云原生应用技术过程的一种机理、流程和理念基础。
+
+## 目标受众
+
+我们的目标受众是希望建设安全的云原生技术生态系统的私营企业、政府机构或非营利组织的首席安全官 (CSO)、首席信息安全官 (CISO) 或首席技术官 (CTO)，也包括组织内的其他相关人员，包括负责设计和实现安全云原生产品服务的项目经理、产品经理、工程师经理和架构师等。除此之外，任何对云原生安全有浓厚兴趣的人都可以参考本文档。
+
+## 云原生目标
+
+容器和微服务架构的采用和革新带来了不少挑战。在现代化组织中，减少网络安全漏洞的需求优先级已经成明显趋势地攀升。同时，随着围绕云应用的创新加速，新的威胁状况也在增加。安全领导层需要通过采取预防、检测和应对网络威胁等措施、满足严格的合规要求，来完成他们身上背负的“保护包括人力 [4] 和非人力资产”任务。然而，有个常见的旧说法，指责这些安全措施阻碍了 DevOps 团队的速度和敏捷性。因此，安全领导层必须搭建起更紧密的集成和互相理解的通道，在为 DevOps 团队赋能更多的同时，使得其能有共同抵御网络风险的责任心。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:69]: 安全领导层 -> 安全行业领袖

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 7]
```diff
@@ -0,0 +1,828 @@
...
+
+_图一_
+
+
+
+云原生技术栈由基础设施层、全生命周期管理、运行环境组成。云原生技术栈可以适用于不同的云计算服务模式：IaaS、PaaS、CaaS 和 FaaS。每种服务部署模式都提供了额外的抽象，以简化云原生环境的管理和操作。由于这些服务模式中的一些模型是众所周知的，并已使用多年，我们将重点关注云原生特有的模型。
+
+CaaS（容器及服务，Containers-as-a-Service) 模式允许用户通过利用基于容器的虚拟化平台、应用编程接口 (API) 或管理页面来管理容器、应用和集群。CaaS 帮助用户构建可扩展的容器化应用，将安全策略嵌入到配置文件中，并在私有云、企业内部数据中心或公有云平台上运行。CaaS 有助于简化构建容器的过程。通过微服务编排和部署，它帮助企业更快地发布软件，并允许在混合云环境之间进行移植，从而降低基础设施投入以及运营成本。CaaS 模式可以帮助企业简化容器管理，同时让企业可以选择只为自己想要和使用的 CaaS 资源付费，从而节约成本。CaaS 以容器为基本资源，而对于 IaaS 环境，则使用虚拟机（VM）和裸金属主机。
+
+FaaS（函数即服务， Functions-as-a-Service ) 是另一种云原生部署模式，是一种允许企业用户通过可执行代码来响应事件的云服务，这种模式用户无需构建和启动微服务相关的复杂基础设施。在云端托管一个软件应用，通常需要配置和管理一个虚拟环境，管理操作系统和网络组件等。通过 FaaS，物理硬件、虚拟机操作系统和 Web 服务器软件管理都由云服务提供商自动处理。从而使用户能够专注于微服务代码中的各个功能，同时为使用的资源付费，并利用云计算提供的资源金进行弹性伸缩。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:99]: 这里不需要空格

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 8]
```diff
@@ -0,0 +1,828 @@
...
+
+• 应该将执行环境限制为特定用户吗？
+
+• 资源的访问权限应该受到限制吗？
+
+• 应该在内核级别限制进程执行吗？
+
+### 容器应用清单扫描
+
+应用程序清单描述了部署容器化应用程序所需的配置。 如基准部分中所述，指南和建议（例如 NIST 800-190 出版物）建议了最佳实践安全实践和应用程序容器的配置。 因此，在 CI/CD 工作流中使用工具来扫描这些应用程序清单以识别可能导致不安全部署状态的配置至关重要。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:181]: 最佳实践安全实践 -> 最佳实践、安全实践

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md [hunk 9]
```diff
@@ -0,0 +1,828 @@
...
+
+#### 仓库隔离
+
+由于使用的开放源码组件往往是从公共来源中提取的。各组织应在其流水线中，创建不同阶段的仓库。只有已授权的开发人员才能访问公共仓库和拉动基础镜像，然后将其存储在内部仓库中，以便在组织内部广泛使用。此外，还建议有单独的私有仓库，用于保存每个团队或小组的开发工件，最后还有一个暂存或预生产仓库，用于准备生产的镜像。这样可以对开源组件的来源和安全性进行更严格的控制，同时可以对 CI/CD 的各个阶段进行不同类型的测试。
+
+对于任何在用的仓库，必须通过专门的认证和权限模型实施访问控制。对所有仓库的连接要采用相互认证的 TLS（以及基础设施内部的其它连接）。
+
+#### 签名、信任和完整性
+
+在构建时对镜像内容进行数字签名，并在使用前对签名数据进行验证，可以保护该镜像数据在构建和运行之间不被篡改，从而确保组件的完整性和来源。首先必需要有流程确认组件是经过审批的。信任确认还包括验证组件是否具有有效的签名。在最简单的情况下，每个组件都可以由一个签名者签名，以表明组件经过了单一的测试和验证过程。然而，软件供应链在大多数情况下比较复杂，创建一个组件依赖于多个验证步骤，因此依赖于实体的信任集团。例如：
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:235]: 必需 -> 必须

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md (hunk 6 above)
Comment (babysor) [reply security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:69]: 这里应该指组织内的安全leaders（例如CSO）吧

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md (hunk 9 above)
Comment (babysor) [reply security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md:235]: Replaced all.

File: security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md (hunk 6 above)
Comment (hbrls) [reply security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md]: 啊，我以为是说业界要赶紧发展通用方案，这样企业才愿意跟进

Comment (babysor) [reply security-whitepaper/cloud-native-security-whitepaper-simple-chinese.md]: 因为段落的第二句说了 “在现代化的组织中”，我读下来是在这个scope内的，你觉得呢？

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md [hunk 10]
```diff
@@ -0,0 +1,826 @@
...
+
+所有的人类和非人类集群以及运维人员都必须要经过认证，他们的所有操作都必须根据访问控制策略进行评估，这些策略将评估每个请求的背景、目的和输出。为了简化认证流程，可以配置联合鉴权，以允许使用公司内的资源，比如多因素认证。然后，必须通过本节提到的访问控制机制进行授权。
+
+#### 凭据管理
+
+##### 硬件安全模块 (HSM)
+
+只要有可能，读者应使用 HSM 等技术，用不离开 HSM 的加密密钥来物理保护加密秘密。如果无法做到这一点，则应使用基于软件的证书管理器。
+
+只要有可能，读者应使用 HSM 技术用加密密钥保护密码学机密算法或信息。如果无法做到这一点，则应使用基于软件的凭据管理器。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:475]: 上面两段是不是重复了

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md [hunk 11]
```diff
@@ -0,0 +1,826 @@
...
+#### 组件仓库
+
+仓库应采用各种技术来对 OCI 组件进行签名和确认。同样重要的是，确保缓存和分发工具也提供签名、加密和提供校验和的能力，以确保缓存层能够检测数据篡改或毒害数据的尝试。
+
+[CNCF 存储白皮书](https://bit.ly/cncf-storage-whitepaperV2) 提供了更多关于云原生存储的概念、术语、使用模式和技术类型的背景资料。
+
+### 访问
+
+#### 身份和访问管理
+
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:440]: 下面这几段好多重复啊

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md [hunk 12]
```diff
@@ -0,0 +1,800 @@
...
+云原生开发可以采用不同的方式来建模，来构成应用开发的生命周期，研发、分发、部署、运行态是其中必不可少的要素之一。云原生安全与传统安全方法的最大不同在于，云原生安全需要有一种强有力的机制，更柔和的嵌入方式，来确保安全措施在不同的环节中有效实施。而不能像传统安全方法一样在应用开发生命周期的末尾，通过分散且粗暴的方式进行干预管理。
+
+同时，我们必须注意到，概念、工具和防护措施的使用和整合需要时间推行，同时需要进行长期的意识教育和培训。否则这些概念、工具和防护措施的采纳和实施可能不会持久，甚至可能出现倒退。
+
+## 生命周期阶段
+
+### 开发
+
+云原生安全强调在应用程序生命周期的早期引入安全工具，安全测试通过尽早识别合规性违规行为和错误配置等安全测试，尽早发现安全问题。具体实现方法就是通过在研发流程中设置安全闸，要求安全问题在流入下一环节前被处置解决。就像处理流水线中的其他问题，如 BugFix、CI Failures 等问题一样，可以尽量缩短问题的修复时间，更好的实现持续改进。
```
Comment (hbrls) [inline security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:27]: 安全测试通过尽早识别合规性违规行为和错误配置等安全测试，
->
通过安全测试尽早识别合规问题和错误配置等，

**Review by hbrls — CHANGES_REQUESTED** [review]

File: security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md [hunk 13]
```diff
@@ -0,0 +1,804 @@
...
+
+#### 安全测试
+
+应用程序和基础设施的自动安全测试应成为安全小组的一个必备重点。测试套件应持续更新，以复制与组织威胁模型相匹配的威胁，并可随着系统的迭代，重复进行安全回归测试。自动化安全测试提高了安全性和发布速度，原因是其取消人工安全校验，如在单一检查点进行验证和人工控制实施，耗时且不充分。自动安全测试还可以通过直接执行一些威胁测试，来证明按需控制的有效性，从而提高系统的安全性，并实时遵守任何内置的合规性要求。
+
+### 工件和镜像
+
+#### 仓库隔离
+
+由于使用的开放源码组件往往是从公共来源中提取的。各组织应在其流水线中，创建不同阶段的仓库。只有已授权的开发人员才能访问公共仓库和拉动基础镜像，然后将其存储在内部仓库中，以便在组织内部广泛使用。此外，还建议有单独的私有仓库，用于保存每个团队或小组的开发工件，最后还有一个暂存或预生产仓库，用于准备生产的镜像。这样可以对开源组件的来源和安全性进行更严格的控制，同时可以对 CI/CD 的各个阶段进行不同类型的测试。
```
Comment (tao12345666333) [inline security-whitepaper/cloud-native-security-whitepaper-simplified-chinese.md:227]: s/拉动/拉取

这样比较符合习惯吧

**Review by lumjjb — APPROVED** [review]
LGTM! This looks really good and looking forward to sharing this with the rest of the community! 
Big thank you to everyone! @rootsongjc, @N3erox0, @cafra, @aiaicaow, @hbrls, @losery, @knwng, @babysor, @gtb-togerther, @dwctua

A couple minor things, which can be included in this PR or a separate one:
- Add a link in https://github.com/cncf/sig-security/blob/master/security-whitepaper/README.md for the translation
- Add a reference to the version of the whitepaper used for translation. The current version is 1.1. Can link to the version in the commit  https://github.com/cncf/sig-security/blob/017e77ff380e303d80adb78e60a1f262e80df0e8/security-whitepaper/cloud-native-security-whitepaper.md

**Review by ashutosh-narkar — APPROVED** [review]

**Pull request merged**